source/install/self-managed-changelog.md

```diff
@@ -5,13 +5,104 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+
+**Release Day: 2021-09-16**
+
+Mattermost v5.39.0 contains a low level security fix. [Upgrading](https://docs.mattermost.com/administration/upgrade.html) to this release is recommended. Details will be posted on our [security updates page](https://mattermost.com/security-updates/) 30 days after release as per the [Mattermost Responsible Disclosure Policy](https://mattermost.org/responsible-disclosure-policy/).
+
+### Improvements
+
+#### User Interface (UI)
+ - Updated in-product help documentation to fix broken links and to correct outdated information.
+ - Italian, Polish, Korean, and Ukrainian languages have been downgraded to alpha. Korean and Ukrainian are [no longer actively maintained](https://handbook.mattermost.com/contributors/join-us/localization#translation-quality).
+
+### Bug Fixes
+ - Fixed a possible panic during license validation.
+ - Fixed an issue with loading of emojis in message attachment titles.
+ - Fixed an issue where the timestamp in deleted messages was not correctly positioned.
+ - Changed the whitespace in the refresh bar so that it always displays to the user.
+ - Fixed an issue where email invites were not sent when clicking the **Next** button during onboarding.
+ - Fixed an issue where clicking "View Message" in an email did not navigate to the post or remember the user's preference to "View in App".
+ - Fixed an issue with the detection of certain collapsible images.
+ - Prevented users from having the unreads filter enabled when the button to toggle it was not shown.
+ - Fixed an issue where Mattermost's shortcut key CTRL+SHIFT+A to open the **Account Settings** clashed with Chrome's CTRL+SHIFT+A that opens a "Search Tabs" pop-up.
+ - Fixed a crash when markdown images were present on the message attachments and embedded bindings.
+ - Fixed an issue that kept message attachment fields unaligned.
+ - Fixed an issue with right-hand side ``SuggestionList`` positioning.
+ - Fixed an issue where Mattermost panicked on ``docx`` files uploaded with ``.doc`` extension.
+ - Fixed a bug with the auto-responder where it would incorrectly calculate the time interval and never send the message.
+ - Fixed a decoding problem for OpenID integration. The requests are now decoded against ``RawURLEncoding``.
+ - Fixed various bugs for the Collapsed Reply Threads (Beta) feature, including:
+   - Fixed an issue where a gap appeared between the first and second consecutive message from the same user.
+   - Fixed an issue where the thread unread state would not update on websocket reconnect.
+   - Fixed an issue where the main channel view root post timestamp added a horizontal scrollbar on hover.
+   - The ``ThreadAutoFollow`` setting is now set to true to enable ``CollapsedThreads``. Previously, this was stated in [the Mattermost documentation](https://docs.mattermost.com/configure/configuration-settings.html#automatically-follow-threads) but was not enforced.
```

Comment:
The ``ThreadAutoFollow`` setting must now be enabled in order to enable ``CollapsedThreads``